### PR TITLE
Page bibliografia

### PR DESCRIPTION
**Pull request by mixcas**
Client asked if the bibliografia could have a wider template.
Also pages didn't have titles.

### CODE REVIEW

**Comment by varffs** [comment]
Pages shouldn't have titles! Thats why they don't

**Comment by mixcas** [comment]
Oh ok, its kinda weird no?

**Comment by varffs** [comment]
No because its worse to duplicate the text in the menu and then directly below in the same formatting. But makes me realise the menu actives were never done

**Comment by mixcas** [comment]
@varffs that makes sense

File: page-bibliografia.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+} else {
+?>
+    <article class="col col-12 u-alert"><?php _e('Sorry, no posts matched your criteria :{'); ?></article>
+<?php
+} ?>
+
+  <!-- end posts -->
+  </section>
+
+  <?php get_template_part('partials/pagination'); ?>
```
Comment (varffs) [inline page-bibliografia.php]: for a single page?

**Comment by varffs** [comment]
Well I kinda think this sucks really as it breaks the design and so on but at this point whatever

**Pull request merged**